<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScenarioEdit</class>
 <widget class="QDialog" name="ScenarioEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item row="4" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>프로젝트</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="EDIT_PROJECT"/>
     </item>
    </layout>
   </item>
   <item row="10" column="0" rowspan="2" colspan="4">
    <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0" columnstretch="0,0,0,0">
     <item row="1" column="1">
      <widget class="QLineEdit" name="EDIT_DATE"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>시험</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QComboBox" name="COMBO_CLASSIFICATION"/>
     </item>
     <item row="1" column="3">
      <widget class="QComboBox" name="COMBO_MODE"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>모드</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>등급</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>날짜</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="EDIT_MISSION"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="BTN_INIT">
       <property name="text">
        <string>시나리오 초기화</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BTN_LOAD">
       <property name="text">
        <string>시나리오 로딩</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BTN_SET">
       <property name="text">
        <string>시나리오 적용</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BTN_SAVE">
       <property name="text">
        <string>시나리오 저장</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>15</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
